src/main/java/XDirectionProject/model/Person.java

```diff
@@ -3,16 +3,16 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
-import javax.persistence.Column;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 
 
```

Comment:
JPA's column annotation is used to create the table properties in the Database. The nullable property of the Column annotation will assure that the properties in the database can't be null.

Reply:
Okey, that`s fine, but if we add also the NonNull annotation provided by Lombok it will check for null value even before we persist to the database, which is more optimal than validation at DB level, my suggestion is to use both.

Reply:
I think validation of the object's properties should be in the DTO's. That way the rest request won't pass if there is a wrong input. I planned this to be added in one of  the next development cycles.

Reply:
That is even better, i agree.